src/app/basic/row-grouping.component.ts

```diff
@@ -133,7 +133,7 @@ import { DataService } from '../data.service';
         <ngx-datatable-column name="Source" prop="source" editable="false" [frozenLeft]="true" />
         <ngx-datatable-column name="Name" prop="name" editable="true" />
         <ngx-datatable-column name="Gender" prop="gender" />
-        <ngx-datatable-column name="Age" prop="age" />
+        <ngx-datatable-column name="Age" prop="age" [hidden]="true" />
```

Comment:
@chintankavathia @timowolf Isn't this just an `ng-if`? I think we should remove this feature again.

Reply:
no it's not ngIf. there are use cases where you want to support sorting on a column which is actually not visible on the view.

Reply:
For those cases we have external sorting.